src/main/java/com/microsoft/azure/functions/SignalRMessage.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.functions;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * <p>
+ * SignalR Message to use with SignalR output binding
+ * </p>
+ *
+ * @since 1.0.0
+ */
+public class SignalRMessage {
+    /**
+     * User to send the message to
+     */
+    public String userId = "";
+
+    /**
+     * Group to send the message to
```

Comment:
Same here, users need to define a POJO I their code.

Reply:
I’d like to include these types unless there is a technical reason not to. Every app that uses these bindings needs these types and I think it’s a bad dev experience to require these to be defined every time. Since we’re moving the annotations to a separate package, we don’t have to worry about adding clutter to the main library.